SecurityManager/Core/Domain/AccessControl/AccessControlSettings.cs

```diff
@@ -0,0 +1,43 @@
+﻿// This file is part of re-strict (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// This program is free software; you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License version 3.0
+// as published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program; if not, see http://www.gnu.org/licenses.
+//
+// Additional permissions are listed in the file re-motion_exceptions.txt.
+//
+using Remotion.ServiceLocation;
+
+namespace Remotion.SecurityManager.Domain.AccessControl
+{
+  [ImplementationFor(typeof(IAccessControlSettings), Lifetime = LifetimeKind.Singleton)]
+  public class AccessControlSettings: IAccessControlSettings
+  {
+    private const bool c_defaultDisableSpecificUser = false;
+
+    public static AccessControlSettings Create (bool? disableSpecificUser = null)
+    {
+      return new AccessControlSettings(disableSpecificUser ?? c_defaultDisableSpecificUser);
+    }
+
+    public bool DisableSpecificUser { get; } = c_defaultDisableSpecificUser;
```

Comment:
I'm not sure it's a good idea to link the default for the factory to the default-ctor's value. they will likely always be the same value (i.e. false) but there's no guarantee that the provvided standard config has every feature switched off. On the factory method, the expectation is that only those features we specify are activated. So, different usecases, no busines logic duplication.